internal/compiler-interface/src/main/contraband/incremental.json

```diff
@@ -103,13 +103,6 @@
           ]
         },
         {
-          "name": "nameHashing",
```

Comment:
I think that to make migration easier we should describe all the breaking changes in a document. But I don't think we have to use deprecations, we're doing a face lift to Zinc so this is the right moment to remove deprecations and do breaking changes. Otherwise we're doomed to maintain the previous, unstable API.